submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -4,40 +4,40 @@
 		<meta charset="UTF-8" >
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" >
 		<title>Document</title>
-		<link rel="stylesheet" href="style.css" >
+		<link rel="stylesheet" href="style.css"> 
 	</head>
 	<body>
 		<div class="wrapper">
 			<div class="header">
 				<nav class="menu">
 					<ul class="menu-list">
 						<li>
-							<a class="menu-link"  href="">Mail</a>
+							<a tabindex="1" class="menu-link"  href="#">Mail</a>
```

Comment:
By default, all interactive elements have a focus, and you can iterate throw them. In your case, you can remove `tabindex` because it will work without them.